react/features/video-menu/components/web/LocalVideoMenuTriggerButton.tsx

```diff
@@ -47,7 +47,7 @@ import TogglePinToStageButton from './TogglePinToStageButton';
  * The type of the React {@code Component} props of
  * {@link LocalVideoMenuTriggerButton}.
  */
-interface Props extends WithTranslation {
+type Props = WithTranslation & {
```

Comment:
why this change?

Reply:
I was testing something and forgot to discard